### PR TITLE
added new routers

### PR DESCRIPTION
**Pull request by IevgenVyshnevskyi**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -7,16 +7,24 @@ function App() {
   const router = createBrowserRouter([
     {
       path: "/",
-      element: <HomePage></HomePage>
+      element: <HomePage/>
```
Comment (FilinBrChaos) [inline src/App.tsx:10]: good job

File: src/App.tsx [hunk 2]
```diff
@@ -7,16 +7,24 @@ function App() {
...
     },
     {
       path: "/categories",
-      element: <CategoriesPage></CategoriesPage>
+      element: <CategoriesPage/>
+    },
+    {
+      path: "/categories/newCategories",
+      element: <CategoriesPage/>
+    },
```
Comment (FilinBrChaos) [inline src/App.tsx:19]: add new route here like (/categories/oldCategories)

**Comment by IevgenVyshnevskyi** [comment]
I did you offer.

File: src/App.tsx (hunk 2 above)
Comment (IevgenVyshnevskyi) [reply src/App.tsx:19]: I did you offer.

Comment (FilinBrChaos) [reply src/App.tsx:19]: cool, all is clear now

**Pull request merged**